content/applications/websites/website/publish/on-premise_geo-ip-installation.rst

```diff
@@ -5,60 +5,55 @@ Geo IP Installation (On-Premises Database)
 Installation
 ============
 
-.. warning::
-   Please note that the installation depends on your computer's operating system and distribution.
-   We will assume that a Linux operating system is being used.
+1. Download both the GeoLite2 City and Country
+   `databases <https://dev.maxmind.com/geoip/geoip2/geolite2/>`_. You should end up with two files
+   called ``GeoLite2-City.mmdb`` and ``GeoLite2-Country.mmdb``
 
-#. Install `geoip2 <https://pypi.org/project/geoip2/>`__ Python library
-    .. code-block:: bash
-
-      pip install geoip2
-
-#. Download the `GeoLite2 City database <https://dev.maxmind.com/geoip/geoip2/geolite2/>`_. You
-   should end up with a file called ``GeoLite2-City.mmdb``
-#. Move the file to the folder ``/usr/share/GeoIP/``
+2. Move the files to the folder ``/usr/share/GeoIP/``
     .. code-block:: bash
 
         mv ~/Downloads/GeoLite2-City.mmdb /usr/share/GeoIP/
+        mv ~/Downloads/GeoLite2-Country.mmdb /usr/share/GeoIP/
 
-#. Restart the server
+3. Restart the server
 
 .. note::
-   If you can't/don't want to locate the geoip database in ``/usr/share/GeoIP/``, you can use the
-   ``--geoip-db`` option of the Odoo command line interface. This option takes the absolute path to
-   the GeoIP database file and uses it as the GeoIP database. For example:
+   If you don't want to locate the geoip database in ``/usr/share/GeoIP/``, use the
+   :option:`--geoip-city-db <odoo-bin --geoip-city-db>` and
+   :option:`--geoip-country-db <odoo-bin --geoip-country-db>` options of the Odoo command line
+   interface. These options take the absolute path to the GeoIP database file and use it as the
+   GeoIP database. For example:
 
    .. code-block:: bash
 
-      ./odoo-bin --geoip-db= ~/Downloads/GeoLite2-City.mmdb
+      ./odoo-bin --geoip-city-db= ~/Downloads/GeoLite2-City.mmdb
 
    .. seealso::
       - :doc:`CLI documentation </developer/misc/other/cmdline>`.
 
-.. warning::
-   ``GeoIP`` Python library can also be used. However this version is discontinued since January
-   1.    See `GeoLite Legacy databases are now discontinued
-   <https://support.maxmind.com/geolite-legacy-discontinuation-notice/>`_
+Test GeoIP Geolocation In Odoo Website
+======================================
+
+Edit a web page to include some geo-ip information such as the country name of the current
+request IP address. To do so:
 
-How To Test GeoIP Geolocation In Your Odoo Website
-==================================================
 1. Go to your website. Open the web page that you want to test ``GeoIP``.
 2. Choose :menuselection:`Customize --> HTML/CSS/JS Editor`.
 3. Add the following piece of XML in the page :
 
 .. code-block:: xml
 
-    <h1 class="text-center" t-esc="request.session.get('geoip')"/>
+    <h1 class="text-center" t-esc="request.geoip.country.name or 'geoip failure'"/>
 
-You should end up with a dictionary indicating the location of the IP address.
+4. Save and refresh the page.
```

Comment:
```suggestion
#. Save and refresh the page.
```
You'll most like need to properly indent the code block above to correctly align it with item 3. as it's currently misaligned.